### PR TITLE
fix(workflow): Render total blacklisted project stats until October 11th

### PR DESCRIPTION
**Pull request by kjlundsgaard**
New granular stats was not displaying the data captured before September 11th, so this displays the filtered events stats available before that time, and more granular stats will display September 11th and onward.

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/projectFilters.jsx [hunk 1]
```diff
@@ -651,9 +662,9 @@ const ProjectFilters = React.createClass({
     let {formattedData} = this.state;
 
     return ReactDOMServer.renderToStaticMarkup(
-      <div style={{width: '150px'}}>
+      <div style={{width: '175px'}}>
         <div className="time-label"><span>{timeLabel}</span></div>
-        <div>{intcomma(totalY)} {totalY > 1 ? t('total events') : t('total event')}</div>
+        <div>{intcomma(totalY)} {totalY != 1 ? t('total events') : t('total event')}</div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectFilters.jsx]: alternatively we could use `tn` from `locale.jsx`:
```javascript
tn('total event', 'total events', totalY)
```

**Review by billyvg — APPROVED** [review]

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectFilters.jsx (hunk 1 above)
Comment (kjlundsgaard) [reply src/sentry/static/sentry/app/views/projectFilters.jsx]: I always forget about that

File: src/sentry/static/sentry/app/views/projectFilters.jsx [hunk 2]
```diff
@@ -395,37 +400,43 @@ const ProjectFilters = React.createClass({
...
+    let cutOverDate = moment([2017, 8, 11]); // date when detailed stats started being recorded
+
     return Object.keys(this.getStatOpts()).map(stat => {
       return {
         data: rawData[stat].map(([x, y]) => {
           if (y > 0) {
             this.setState({blankStats: false});
           }
+
+          //TODO(maxbittker) this is only needed until October 10th, 2017 :
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectFilters.jsx]: here

**Pull request merged**